knowledge_graphs/knowledge_graph_validator_composed.py

```
@@ -49,261 +50,120 @@ def __init__(
+            result.summary = "Validation failed due to error"
+
+        return result
+
+    async def _validate_imports(self, imports: list[ImportInfo]) -> list[ImportValidation]:
+        """Validate all imports using the import validator."""
+        validations = []
+        for import_info in imports:
+            try:
+                validation = await self.import_validator.validate_import(import_info)
+                validations.append(validation)
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)